leku/build.gradle

```diff
@@ -139,7 +177,7 @@ bintray {
     description = "Location picker component for Android. It returns a latitude,longitude and an address based on the location picked in the LocationPickerActivity provided."
     publish = true
     dryRun = false
-    override = true
+    override = false
```

Comment:
Because we don't want to override an AAR already published.

This is what Docu says:

> override = false //[Default: false] Whether to override version artifacts already published

Reply:
Then it should have never been true.
It would be better to remove the line and keep it default.

Reply:
@CristianGM done! You're right it never should have been true but it was hard to make it work without testing these things.